src/container/plugins/FocusPlugin.js

```diff
@@ -135,15 +136,23 @@
 		// we will pause the game until a blur event to that item
 		// has been sent
 		var self = this;
-		$(this.options.pauseFocusSelector).on('focus', function()
-		{
-			self._isManualPause = self.paused = true;
-			$(this).one('blur', function()
+		document
+			.querySelector(this.options.pauseFocusSelector)
```

Comment:
This might introduce a bug, because `this.options.pauseFocusSelector` could match nothing (and return `null`) which'll make that `addEventListener` call fail.

Reply:
Would that not correct result if they gave it an invalid element? Or do you want it to silently fail? That could get really annoying to debug if that is the case.

Reply:
True. I guess another option would be to still do the check, but throw a _new_ error with better messaging. Currently the error would look like "undefined is not a function". But with this code:

```javascript
var element = document.querySelector(this.options.pauseFocusSelector);
if(element === null) {
  throw new Error('Pause focus selector returns no elements'); // something like that
}
element.doImportantStuff(some, parameters);
```